front/app/components/SignUpIn/AuthProviderButton.tsx

```diff
@@ -99,6 +99,7 @@ const ContinueButton = styled(Button)``;
 
 interface Props {
```

Comment:
This should also be exported. Normally, there should be an error in your code editor. Also make sure to run the full CI checks on this branch. They would have caught this. Only a few seem to have run.

Reply:
How can I make sure that all frontend tests are being run @brentguf?

Reply:
You can trigger it it manually with:
```

curl --request POST \
  -u 77f63cf7e8af4cd03a3c8391280dfc9bed99dd07: \
  --url https://circleci.com/api/v2/project/github/CitizenLabDotCo/citizenlab/pipeline \
  --header 'content-type: application/json' \
  --data '{"branch": "CL2-6937_omniauth_vienna_saml","parameters": {"trigger": false, "back": false, "front": true}}'
```

Reply:
This is what I did already, are sure it should have errored? It does not throw any errors on my local machine either.

Reply:
Maybe only related to the changes I'm suggesting, so if you make these, it should pop up. :-)

Reply:
Checks are running now in any case!

Reply:
It is green. So do I have to export the Props? :thinking:

Reply:
Then no... But with the solution I'm suggesting for the icon prop, it did raise an error with me. Just assign me again to review when you're done and I'll have a look.